README

```diff
@@ -13,16 +13,16 @@ To get started using Apache Trafodion:
 
   - documentation for this release can be viewed on the Apache Trafodion 
     wiki page: 
-      http://trafodion.incubator.apache.org/documentation.html
+      http://trafodion.apache.org/documentation.html
 
   - release notes describing changes for particular releases can be viewed:
       http://trafodion.apache.org/release-notes.html
 
-  - the latest Apache Trafodion can be downloaded from Apache incubator
+  - the latest Apache Trafodion can be downloaded from Apache
```

Comment:
Grammar nit: Do you want to say "the Apache distribution site" (add "the") ?

Reply:
Okay, I will add "the"